README.md

```diff
@@ -537,6 +537,7 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
     - plastid
       - 'ddAraThal4.HiFi.reads.fasta': test dataset for plastid reads for Arabidopsis thaliana
   - saccharomyces_cerevisiae
+    - samplesheet.csv: sample sheet as used by the nf-core/rnaseq test profile
```

Comment:
can you update the comment?

Reply:
This is there to highlight where the data comes from